scripts/build/termux_step_start_build.sh

```diff
@@ -47,6 +49,28 @@ termux_step_start_build() {
 		fi
 	fi
 
+	echo "termux - building $TERMUX_PKG_NAME for arch $TERMUX_ARCH..."
+	test -t 1 && printf "\033]0;%s...\007" "$TERMUX_PKG_NAME"
+
+	# Avoid exporting PKG_CONFIG_LIBDIR until after termux_step_host_build.
+	export TERMUX_PKG_CONFIG_LIBDIR=$TERMUX_PREFIX/lib/pkgconfig
+
+	if [ "$TERMUX_PKG_BUILD_IN_SRC" = "true" ]; then
+		echo "Building in src due to TERMUX_PKG_BUILD_IN_SRC being set to true" > "$TERMUX_PKG_BUILDDIR/BUILDING_IN_SRC.txt"
```

Comment:
Take out this one `echo...` line and it will work.

Reply:
Oh, change this check to also only run when continuing, or move it inside the next block.

Reply:
Moved the removal and creation of the folders to a new function termux_step_setup_build_folders that is run before termux_step_start_build. That way we only need to have the `if [ "$TERMUX_PKG_BUILD_IN_SRC" = "true" ]` check in only one place, in termux_step_start_build. Verified to work with zip (which builds in src), and nim (which has a hostbuild step)